os_migrate/tests/unit/fixtures.py

```diff
@@ -133,3 +133,92 @@ def serialized_network():
         },
         const.RES_TYPE: 'openstack.network.Network',
     }
+
+
+def sdk_security_group():
+    return openstack.network.v2.security_group.SecurityGroup(
+        description='Default security group',
+        location='location',
+        cloud='devstack',
+        project='project',
+        region_name='RegionOne',
+        zone='null',
+        name='default',
+        revision_number='1',
+        security_group_rules=[],
+        tags=[],
+        created_at='2020-01-30T14:49:06Z',
+        project_id='uuid-project',
+        tenant_id='uuid-tenant',
+        updated_at='2020-01-30T14:49:06Z',
+    )
+
+
+def serialized_security_group():
```

Comment:
Heads up for later when editing: here we should also drop things like `location` and `project`, move `revision_number` to info etc. -- make it in sync with our serialization.

Reply:
Still TBD